### PR TITLE
Game statics class

### PR DESCRIPTION
**Pull request by ILyell**
Add game statics methods and test to stat_tracker class.

### CODE REVIEW

**Comment by ParkerBoeing** [comment]
Good bolstering of the feature csvs. Great work to all on finishing game stat methods!

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -25,4 +25,60 @@ def from_csv(locations_hash)
...
+
+  def count_of_games_by_season
+  season_games = @games.each_with_object(Hash.new(0)) {|game, hash| hash[game.season] += 1}
+  p season_games
+  end
+
+  def highest_total_score
+    highest_score = 0
+    @games.each do |game|
+      # binding.pry
```
Comment (ParkerBoeing) [inline lib/stat_tracker.rb:44]: Note to future selves - we'll remove this pry

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -25,4 +25,60 @@ def from_csv(locations_hash)
       @game_by_team << Game_By_Team.new(row)
     end
   end
+      
+#---------Game Statics Methods-----------
+  def percentage_ties
+    tie_count = @games.count { |game| game.away_goals.to_i == game.home_goals.to_i }
+    percentage = (tie_count.to_f / @games.count.to_f).round(4) * 100
+    p percentage
+  end
```
Comment (DylanScotty) [inline lib/stat_tracker.rb:34]: test comment

File: lib/stat_tracker.rb [hunk 3]
```diff
@@ -25,4 +25,60 @@ def from_csv(locations_hash)
...
+  def lowest_total_score
+    lowest_score = nil
+    @games.each do |game|
+      total_score = game.home_goals.to_i + game.away_goals.to_i
+      lowest_score = total_score if lowest_score.nil? || total_score < lowest_score
+    end
+    lowest_score
+  end
+
+  def percentage_visitor_wins
```
Comment (ParkerBoeing) [inline lib/stat_tracker.rb:60]: can make percentage home_wins and percentage away_wins look more similar when we refactor as they are basically the same process

Comment (MatthewTLim) [reply lib/stat_tracker.rb:60]: great idea maybe we can make a helper for this!

**Pull request merged**